docs/reference/commandline/service_create.md

```diff
@@ -915,6 +916,17 @@ Supported isolation modes on Windows are:
 - `process`: use process isolation (Windows server only)
 - `hyperv`: use Hyper-V isolation
 
+### Create services requesting Generic Resources
+
+You can narrow the kind of nodes your task can land on through the using the
+`--generic-resources` flag (if the nodes advertise these resources):
```

Comment:
`s/generic-resources/generic-resource/`

also 2 instances in the example below

Reply:
Fixed, also fixed in `--generic-resources-add` and `--generic-resources-rm` to ``--generic-resource-add` and ``--generic-resource-rm`